user-interface/src/case-detail/panels/CaseDetailBasicInfo.tsx

```diff
@@ -208,26 +221,6 @@ export default function CaseDetailBasicInfo(props: CaseDetailBasicInfoProps) {
         </div>
         <div className="additional-debtor-information padding-bottom-4 case-card">
           <h3>Additional Debtor Info</h3>
-          {props.caseDetail.debtor.taxId && (
-            <div
-              data-testid="case-detail-debtor-taxId"
-              aria-label="debtor employer identification number"
-            >
-              <span className="case-detail-item-name">EIN:</span>
-              <span className="case-detail-item-value">{props.caseDetail.debtor.taxId}</span>
-            </div>
-          )}
-          {props.caseDetail.debtor.ssn && (
-            <div data-testid="case-detail-debtor-ssn" aria-label="debtor social security number">
-              <span className="case-detail-item-name">SSN/ITIN:</span>
-              <span className="case-detail-item-value">{props.caseDetail.debtor.ssn}</span>
-            </div>
-          )}
-          {!props.caseDetail.debtor.taxId && !props.caseDetail.debtor.ssn && (
-            <div data-testid="case-detail-debtor-no-taxids" aria-label="debtor tax identification">
-              {informationUnavailable}
-            </div>
-          )}
         </div>
```

Comment:
There's nothing left in this div other than the header. I'd advocate for removing it, or at least hiding it if there is nothing to display in the section.